src/components/TopNavigation/index.js

```diff
@@ -42,17 +44,25 @@ const TopNavigation = () => {
             <li>
               <Link
                 to='/items'
-                className={location.pathname === '/items' ? 'activ-nav' : ''}
+                className={isItemsPage ? 'activ-nav' : ''}
+                style={{ color: isAddItemPage ? '#3692FF' : '' }}
               >
                 중고마켓
               </Link>
             </li>
           </ul>
         </nav>
       </div>
-      <Link to='/login'>
-        <Button title='로그인' />
-      </Link>
+      {isAddItemPage && (
+        <div className='rightContainer'>
+          <img src={pandaProfile} alt='현재 프로필' className='profileIcon' />
+        </div>
+      )}
+      {!isAddItemPage && (
+        <Link to='/login'>
+          <Button title='로그인' />
+        </Link>
+      )}
```

Comment:
삼항연산자 사용이 가능한 곳처럼 보이네요! 

```suggestion
      {isAddItemPage ? (
        <div className='rightContainer'>
          <img src={pandaProfile} alt='현재 프로필' className='profileIcon' />
        </div>
      : (
        <Link to='/login'>
          <Button title='로그인' />
        </Link>
      )}
```